### PR TITLE
Bug fix for sending metrics from frontend

### PR DESCRIPTION
**Pull request by dineshg13**
Bug fix for frontend , to make sure it sends metrics. Before this change , we were initialing `meter` and `httpRequestCounter` before we set `global`  meterProvider. 

## Changes
After this change, when we run `docker compose up` , we should be able to see metrics in `prometheus` .

### CODE REVIEW

File: src/frontend/main.go [hunk 1]
```diff
@@ -114,7 +114,7 @@ func InitMeterProvider() (*controller.Controller, error) {
 		return nil, err
 	}
 
-	controller := controller.New(
+	pusher := controller.New(
```
Comment (dineshg13) [inline src/frontend/main.go:117]: This change is to avoid overriding controller which package that is being imported.

**Comment by cartersocha** [comment]
@fatsheep9146, can you take a look at this please?

**Comment by fatsheep9146** [comment]
> @fatsheep9146, can you take a look at this please?

Absolutely, I will look at this pr

**Comment by fatsheep9146** [comment]
This pr is lgtm. @cartersocha 
But there exists conflicts to be resolved @dineshg13

**Review by cartersocha — APPROVED** [review]

**Pull request merged**